src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -640,110 +640,84 @@
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\CultureData.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\CultureInfo.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\CultureNotFoundException.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\CultureTypes.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DaylightTime.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DateTimeFormat.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DateTimeFormatInfo.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DateTimeFormatInfoScanner.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DateTimeParse.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DateTimeStyles.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\DigitShapes.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\EncodingDataItem.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\EncodingTable.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\GlobalizationAssembly.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\EastAsianLunisolarCalendar.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\GregorianCalendar.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\GregorianCalendarTypes.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\GregorianCalendarHelper.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\GregorianCalendarTypes.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\HebrewCalendar.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\HebrewNumber.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\HijriCalendar.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\UmAlQuraCalendar.cs" />
-    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\PersianCalendar.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\IdnMapping.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\InternalGlobalizationHelper.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\JapaneseCalendar.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\JapaneseLunisolarCalendar.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\JulianCalendar.cs" />
     <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\KoreanCalendar.cs" />
+    <MscorlibSources Include="$(BclSourcesRoot)\System\Globalization\KoreanLunisolarCalendar.cs" />
```

Comment:
Shouldn't we be rather keeping the version of these files from under corefx? They version under corefx is better formatted and factored.